### DISCUSSION TITLE
English example sentence for 駄々をこねる is incorrect/weird

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#49] [Errors and Corrections]
JP: あの子はいつもだだをこねている

EN: That child is always fretting.

---

This usage of "fretting" feels weird to me in English. I've asked a decent sample of native speakers about it and they all said it doesn't feel right. Checking various dictionary definitions for the verb "to fret" don't seem to give me any results of this specific usage. The expression  駄々をこねる in Japanese refers to something like a child throwing a tantrum or voicing unreasonable complaints in a childish/spoiled manner (子どもが親に甘えてわがままをいったり、すねたりすること). I don't think "fret" carries this nuanced meaning.

A better English translation would be something like "That child is always throwing tantrums".

**Comment by stephenmk** [comment]
Thanks, I updated it to use the new translation that you suggested.
